vue/sbc-common-components/src/components/SbcSignin.vue

```diff
@@ -62,24 +62,8 @@ export default class SbcSignin extends NavigationMixin {
           // sync the account if there is one
           await this.syncAccount()
 
-          // redirect to create account page if the user has no 'account holder' role
-          const isRedirectToCreateAccount = (userInfo.roles.includes(Role.PublicUser) && !userInfo.roles.includes(Role.AccountHolder))
-          if (isRedirectToCreateAccount) {
-            switch (userInfo.loginSource) {
-              case LoginSource.BCSC:
-                this.redirectToPath(this.inAuth, Pages.CHOOSE_AUTH_METHOD)
-                break
-              case LoginSource.BCEID:
-                this.redirectToPath(this.inAuth, Pages.NON_BCSC_INSTRUCTIONS)
-                break
-            }
-          }
-
           // emit an event to parent that user profile syncing is completed.
-          // the argument isRedirectToCreateAccount will be used to check in the parent signin view component
-          // to determine whether to perform the redirect on that end,
-          // if the value is true, then the redirection should be happening from this component.
-          this.$emit('sync-user-profile-ready', isRedirectToCreateAccount)
+          this.$emit('sync-user-profile-ready')
```

Comment:
What is the change here ? If the user is not on accounts application, how is this going to work ? I think this change breaks the whole redirection you did as part of previous ticket to redirect if the user is not part of any account. Please correct if I am wrong.

Reply:
Yeah, but actually we are handling the redirection in our auth app.
User will come to the auth app anyways for the signin process.
and the redirectUrl prop from them is actually making the redirection back to their app.
I have discussed with saravan and intercepted that part and made the changes here. which will be much better maintainable than this.

https://github.com/bcgov/sbc-auth/pull/870

this PR is having the changes